### DISCUSSION TITLE
Add "IN" field for Education section in locale_catalog

### DISCUSSION THREAD

**Discussion by 404mat** [rendercv/rendercv#216] [General]
RenderCV is a great tool ! This and the pipeline are now saving me countless time.

One thing I would like to be added : maybe I did not search enough in the [documentation](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file/?h=locale_catalog#locale_catalog-field), but as far as I know, the word `in` in the Education section is always displayed in English.

For reference, the following picture displays my education section written in French, except the word `in` between the degree name and the program :
![image](https://github.com/user-attachments/assets/24ac68d9-e1e3-4ef1-a108-50987e8fffd0)

It could be implemented to be used like the `to` keyword for the dates :
```
locale_catalog:
    to: "-"
    in: "en"
```
But it could be confusing. For example in French, `in` could be translated to `en` or `dans` (`in` or `inside`, respectively). Even though `inside` is not used in the templates, if we ever want to eliminate the ambiguity, we could use something like : 
```
locale_catalog:
    to: "-"
    education_in: "en"
```

It is a rather small and low priority suggestion so i understand if you can't work on it now.

Thanks a lot for the amazing work so far !

**Comment by sinaatalay** [comment]
Thank you for the suggestion! We should have a look at this.

Meanwhile, you can change “in” to anything by updating the `EducationEntry.j2.tex` file (created by the `rendercv new` command). See [here](https://docs.rendercv.com/user_guide/#overriding-built-in-themes) for details.

**Comment by 404mat** [comment]
Good, I think that is what I will do as an alternative 👍 It is harder to do in the context of a pipeline but I will try and add the logic to handle this. Thanks !

**Comment by sinaatalay** [comment]
It’s actually very easy; there isn’t a need for any logic. You should just update [this file](https://github.com/sinaatalay/rendercv-pipeline/blob/main/src/classic/EducationEntry.j2.tex) in the pipeline.

**Comment by 404mat** [comment]
My situation is I have modified the original pipeline that you provided to generate CVs in 2 languages (french and english). If I update the file you mentioned, I'm afraid I will be in a situation where my english resume will be generated with a french word inside it.
Maybe an alternative is to create a "custom theme" for each file to use that are the same except for the `EducationEntry.j2.tex` file.

Edit: I think I forgot to say it at first, this happens in the `engineeringresumes` theme !

**Comment by sinaatalay** [comment]
Then, I would suggest using arbitrary keys to dynamically change the “in” keyword from the YAML file. See [here](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file/#using-arbitrary-keys) for details.
